src/docs/api/concepts.mdx

```diff
@@ -95,3 +95,49 @@ GET /api/0/projects/{project_slug}/?collapse=stats
 ```
 
 This is typically only needed if the endpoint is already public and we do not want to introduce a breaking change. Remember, if the endpoint is public and we remove an attribute, it is a breaking change. If you are iterating on an undocumented endpoint, return the minimal set of attributes and rely on the `expand` parameter to get more detailed information.
+
+## Paginating responses
+>> APIs often need to provide collections of data, most commonly in the `List` standard method.
+However, collections can often be arbitrarily sized, and also often grow over time, increasing lookup time as well as the size of the responses being sent over the wire.
+Therefore, it is important that collections be paginated.
+
+Paginating responses is a [standard practice for APIs](https://google.aip.dev/158), and we follow them at Sentry as well.
+We've seen an example of a `List` endpoint above, and these endpoints have two tell-tale signs:
+```json
+GET /api/0/projects/{project_slug}/teams
+[
+  {
+    "id": 1,
+		"name": "Team 1",
+		"slug": "team1",
+  },
+	{
+    "id": 2,
+		"name": "Team 2",
+		"slug": "team2",
+  }
+]
+
+```
+1. The endpoint returns an array, or multiple, objects instead of just one.
+2. The endpoint can sometimes end in a plural (s), but more importantly, it does __not__ end in an identifier (`*_slug`, or `*_id`).
+
+To paginate a response at Sentry, you can leverage the [`self.paginate`](https://github.com/getsentry/sentry/blob/24.2.0/src/sentry/api/base.py#L463-L476) method as part of your endpoint.
```

Comment:
Can you add something around the line that self.paginate is the standardized way we paginate at sentry which helps us with unification of logging and monitoring?

Reply:
yeah sure!

Reply:
done!